src/index.tsx

```diff
@@ -63,17 +63,19 @@ ReactDOM.render(
     <FixedGlobalStyle />
     <Web3ReactProvider getLibrary={getLibrary}>
       <Web3ProviderNetwork getLibrary={getLibrary}>
-        <Blocklist>
-          <Provider store={store}>
-            <Updaters />
-            <ThemeProvider>
-              <ThemedGlobalStyle />
-              <HashRouter>
-                <App />
-              </HashRouter>
-            </ThemeProvider>
-          </Provider>
-        </Blocklist>
+        <LanguageProvider>
```

Comment:
should it be at the top just below strict mode? doesn't depend on anything else, and theoretically the theme might depend on it

Reply:
will move below strict mode for now and consider integrating with redux in a follw-up cl